tests/e2e_test.py

```diff
@@ -110,9 +118,13 @@ def test_dune_results_to_db(self):
             types,
         )
 
+        self.assertTrue(isinstance(dynamic_type, NUMERIC))
+        self.assertEqual(dynamic_type.precision, 12)
+        self.assertEqual(dynamic_type.scale, 7)
+
         pg.save((df, types))
 
-        self.assertEqual(
+        self.assertListEqual(
```

Comment:
Not sure what the question is about. We're comparing lists so in case of a difference, ListEqual would give a more useful diff. Or were you asking about something else?

Reply:
Nah I've just never seen this assertion type before. Cool stuff.